tests/publiccloud/run_ltp.pm

```diff
@@ -151,7 +151,8 @@ sub run {
     $instance->run_ssh_command(cmd => 'sudo CREATE_ENTRIES=1 ' . get_ltproot() . '/IDcheck.sh', timeout => 300);
     record_info('Kernel info', $instance->run_ssh_command(cmd => q(rpm -qa 'kernel*' --qf '%{NAME}\n' | sort | uniq | xargs rpm -qi)));
     if (get_var('PUBLIC_CLOUD_INSTANCE_TYPE') =~ /-metal$/) {
-        record_info('VM type', $instance->run_ssh_command(cmd => '! systemd-detect-virt'));
+        # The metal detector fails on GCP because it may return "google"
+        record_info('VM type', $instance->run_ssh_command(cmd => '! systemd-detect-virt')) unless is_gce;
```

Comment:
```suggestion
        record_info('VM type', $instance->run_ssh_command(cmd => 'systemd-detect-virt || true'));
```
This this call is only printing the output, I wonder if it makes even sense to run `! systemd-detect-virt` and expect it to fail.

Printing the output of a command one expects to fail look hacky and I'm not sure if it just works by accident or by intent.

Reply:
We want _both_ the status & the output.

Reply:
Ok, I understand.